src/components/shop/shop.js

```diff
@@ -17,8 +21,43 @@ export class Shop extends Screen {
 
     create() {
         this.addBackgroundItems();
-        const buttons = ["exit", "audio"];
-        this.setLayout(buttons);
-        this.panel = scrollableList(this);
+        this.setLayout(["home", "pause"]);
+        const metrics = getMetrics();
+        this.title = this.createTitle(metrics);
+        this.wallet = createWallet(this, metrics);
```

Comment:
Is there a better name than wallet?
Wallet implies it holds the currency and deals with transactions where as this is just a ui display of currency.

I admit I am struggling to think of anything specific enough myself. Might be worth floating in game-dev.

Reply:
Following chat, renamed `wallet` to `balance` across the PR. d856e23